src/modules/ekf2/EKF2.hpp

```diff
@@ -124,14 +124,21 @@ class EKF2 final : public ModuleParams, public px4::ScheduledWorkItem
 	static bool trylock_module() { return (pthread_mutex_trylock(&ekf2_module_mutex) == 0); }
 	static void unlock_module() { pthread_mutex_unlock(&ekf2_module_mutex); }
 
-	bool multi_init(int imu, int mag);
+	bool multi_init(int imu, int mag, uint8_t vis_mode);
 
 	int instance() const { return _instance; }
 
 private:
 
 	static constexpr uint8_t MAX_NUM_IMUS = 4;
 	static constexpr uint8_t MAX_NUM_MAGS = 4;
+	static constexpr uint8_t MAX_NUM_VIS_GNSS = 2;
+
+	static constexpr uint8_t VIS_NORMAL_OPERATION = 0;
```

Comment:
Since these constants are defined both here and in the msg-file, it would be nice to have a short comment/reference to eachother.

Reply:
(or possible do not define at all in msg?)

Reply:
Should probably just define it in the msg and use those everywhere.

Reply:
Using the msg definitions now